all/generate/build.sh

```diff
@@ -124,7 +124,8 @@ cp .openapi-generator-ignore output/.openapi-generator-ignore
 mv $sdk_name.json output/$sdk_name.json
 
 # build the sdk
-docker-compose build && docker-compose run lusid-sdk-gen ./generate/generate.sh ./generate ./generate/output $swagger_file $config_file 
+docker-compose build && docker-compose run lusid-sdk-gen ./generate/generate.sh ./generate ./generate/output \
+    $(basename $swagger_file) $(basename $config_file)
```

Comment:
Just a question, what is basename here and where is it specified?

Reply:
basename takes the "filename" from the path ... i.e. in this case is trips the path used as source of copy ... the copy is to the working folder so the call to docker does not need the path specified.